### PR TITLE
Add minor polishes to userGuide

### PR DESCRIPTION
**Pull request by raysonkoh**
**What is the purpose of this pull request?**

- [x] Documentation update
- [ ] Bug fix
- [ ] Feature addition or enhancement
- [ ] Code maintenance
- [ ] Others, please explain:

Two main fixes:
- Fix broken url fragments in `deployingTheSite`, thanks @wxwxwxwx9 for bringing this up.
- Add link to code block copy button plugin in `formattingContents`->`code` for better visibility of the plugin

**Anything you'd like to highlight / discuss:**

**Testing instructions:**

**Proposed commit message: (wrap lines at 72 characters)**
Add minor polishes to the userGuide

---

**Checklist:** :ballot_box_with_check:

- [x] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [x] Linked all related issues
- [x] No blatantly unrelated changes
- [x] Pinged someone for a review!

### CODE REVIEW

File: docs/userGuide/deployingTheSite.md [hunk 1]
```diff
@@ -21,15 +21,15 @@
 
 ## Generic steps for deploying a MarkBind site
 
-1. Set the [`baseUrl` property of the `site.json` file](siteJsonFile.html#baseUrl) to match the deploy location.
+1. Set the [`baseUrl` property of the `site.json` file](siteJsonFile.html#baseurl){.no-convert} to match the deploy location.
```
Comment (jonahtanjz) [inline docs/userGuide/deployingTheSite.md:24]: Is there a reason why `no-convert` is needed here? I tried removing it and the link works fine without it.

**Review by jonahtanjz — COMMENTED** [review]
Nice work 👍 Just one question, other than that everything else looks good :)

File: docs/userGuide/deployingTheSite.md (hunk 1 above)
Comment (raysonkoh) [reply docs/userGuide/deployingTheSite.md]: It shouldn't be there. Thanks for the catch!

File: docs/userGuide/syntax/code.mbdf [hunk 2]
```diff
@@ -157,6 +157,16 @@ You can also use multiple features together, as shown below.
...
 
+##### Copy button
+
+A _copy_ button can be added to code blocks using the `codeBlockCopyButtons` plugin:
+
+<panel type="seamless" header="**User Guide: Using Plugins → Plugin: `codeBlockCopyButtons`**" popup-url="usingPlugins.html#plugin-codeblockcopybuttons">
+
+  <include src="../plugins/codeBlockCopyButtons.mbdf" />
+
+</panel>
```
Comment (ang-zeyu) [inline docs/userGuide/syntax/code.mbdf:168]:
```suggestion
</panel>
<br>
```

a little squeezy 🙃

**Review by ang-zeyu — COMMENTED** [review]
Lgtm 👍 just one minor nit:

**Comment by raysonkoh** [comment]
Thanks for the review @ang-zeyu 😄 , I have made the change that you requested

**Review by ang-zeyu — APPROVED** [review]

**Pull request merged**